src/js/va-letters/containers/DownloadLetters.jsx

```diff
@@ -1,18 +1,44 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
+// TODO: style this properly once design is settled
 class DownloadLetters extends React.Component {
   render() {
+    const destination = this.props.destination || {};
+    const letterItems = (this.props.letters || []).map((letter) => {
+      return (
+        <li key={letter.letterType}>
+          <a href="#">{letter.letterName}</a>
+        </li>
+      );
+    });
     return (
-      <div>Placeholder for download letters</div>
+      <span>
+        <div className="letters-form-panel">
+          <p>The address on file for you with VA Compensation and Pension is:</p>
+          <span>{destination.fullName}</span><br/>
+          <span>{destination.addressLine1}, {destination.addressLine2} {destination.addressLine3}</span><br/>
+          <span>{destination.city}, {destination.state} {destination.zipCode}</span>
+        </div>
+        <div>
+          <p>The following letters are available for download:</p>
+          <ul>
+            {letterItems}
+          </ul>
+        </div>
+      </span>
     );
   }
 }
 
 function mapStateToProps(state) {
   const userState = state.user;
+  const letterState = state.letters;
   return {
-    profile: userState.profile
+    profile: userState.profile,
+    letters: letterState.letters,
+    destination: letterState.destination,
+    available: letterState.available
```

Comment:
Oh, I just realized the `available` here is unrelated to the EVSS values...I think I just added this so that if the data fetch failed, we have some indication of it and can update the UI appropriately.